src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -261,9 +261,21 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
     const isTypeTrackExpense = iouType === CONST.IOU.TYPE.TRACK_EXPENSE;
     const canEditDistance = isTypeRequest || (canUseP2PDistanceRequests && isTypeSplit);
 
-    const {unit, rate, currency} = mileageRate;
+    // if there is no policyID that means that the transaction was started from Global Create and the participant is not a policy
+    const personalPolicy = policyID === CONST.POLICY.ID_FAKE || !policyID ? PolicyUtils.getPersonalPolicy() : policy;
```

Comment:
My general understanding is that we should get all Onyx values from withOnyx rather than using a util function, so that the component updates when the value updates in Onyx. That's why some of the existing utils are marked deprecated.

Reply:
It's also confusing to call this personalPolicy if it can fallback on policy, which will not be personal.